lib/spack/docs/packaging_guide.rst

```diff
@@ -4016,8 +4016,11 @@ the package being tested when ``installed`` is ``True``.
 The executable runs in ``work_dir``, when specified, using the provided
 ``options``. The return code is checked against the ``status`` argument,
 which can be an integer or list of integers representing status codes
-corresponding to successful execution. Spack also checks that every string
-in ``expected`` is a regex matching part of the output from the test run.
+corresponding to successful execution (e.g. ``status=[0,3,7]``).
+Spack also checks that every string in ``expected`` is a regex matching
+part of the output from the test run (e.g.
+``expected=['completed succesfully', 'converged in']``). Default behavior
+is to behave as though ``status=[0,]`` is specified.
```

Comment:
Maybe a nit, but why include "," in the list?  (I'm used to seeing inclusion of a comma when one is initializing a set, not a list.)

Reply:
Maybe my python ignorance. Thought it was needed for lists too. Will fix